### PR TITLE
test/tablets: Add test to check how ALTER changes RF (in one DC)

### PR DESCRIPTION
**Pull request by xemul**
For now test is incomplete in several ways

1. It xfails, until #17116
2. It doesn't rebuild/repair tablets
3. It doesn't check that tablet data actually exists on replicas

refs: #17575

### CODE REVIEW

File: test/topology_custom/test_tablets.py [hunk 1]
```diff
@@ -95,3 +96,45 @@ async def test_reshape_with_tablets(manager: ManagerClient):
...
+    if direction == 'down':
+        rf_from = 3
+        rf_to = 2
+
+    await cql.run_async(f"CREATE KEYSPACE test WITH replication = {{'class': 'NetworkTopologyStrategy', '{this_dc}': {rf_from}}}")
+    await cql.run_async("CREATE TABLE test.test (pk int PRIMARY KEY, c int);")
+
+    logger.info("Populating table")
+    for _ in range(32):
+        await asyncio.gather(*[cql.run_async(f"INSERT INTO test.test (pk, c) VALUES ({k}, {k});") for k in range(64)])
```
Comment (tgrabiec) [inline test/topology_custom/test_tablets.py:125]: Why do it 32 times? The batch is the same in each iteration.

Comment (xemul) [reply test/topology_custom/test_tablets.py:125]: Yikes. It was 32 tables in the test initially, then I left only one and forgot to clean this place. Will fix

File: test/topology_custom/test_tablets.py [hunk 2]
```diff
@@ -95,3 +96,45 @@ async def test_reshape_with_tablets(manager: ManagerClient):
...
+    res = await cql.run_async("SELECT data_center FROM system.local")
+    this_dc = res[0].data_center
+
+    if direction == 'up':
+        rf_from = 2
+        rf_to = 3
+    if direction == 'down':
+        rf_from = 3
+        rf_to = 2
+
```
[the remark this answers is not in the document]
Comment (xemul) [reply test/topology_custom/test_tablets.py:122]: Good point

**Comment by xemul** [comment]
upd:
- populate table without loop
- add case for RF change

**Pull request closed without merging**